documentation/missionmaker/modules.md

```diff
@@ -11,34 +11,37 @@ parent: wiki
 ### 1.1 Advanced Ballistics
 *Part of: ace_advanced_ballistics*
 
-This module allows to enable advanced external- and internal ballistics.
+This module allows enabling and configuring advanced ballistic simulations.
 
 **Settings:**
 
 1. **Advanced Ballistics (Boolean)**<br>
 Enables advanced ballistics.<br>
 `Default value: No`
-2. **Always Enabled For Snipers (Boolean)**<br>
-Always enables advanced ballistics when high power optics are used.<br>
+2. **Enabled For Snipers (Boolean)**<br>
+Enables advanced ballistics for non local snipers (when using high power optics).<br>
 `Default value: Yes`
-3. **Disabled In FullAuto Mode (Boolean)**<br>
+3. **Enabled For Group Members (Boolean)**<br>
+Enables advanced ballistics for non local group members.<br>
+`Default value: No`
+4. **Enabled For Everyone (Boolean)**<br>
+Enables advanced ballistics for all non local players (Enabling this feature may degrade performance during heavy firefights in multiplayer).<br>
```

Comment:
That's a good point. I'll check all the documents for parentheses that have a word starting with a capital letter where it shouldn't. Moment.